src/RestHandler.cpp

```diff
@@ -31,7 +31,7 @@ using web::http::status_codes;
 using web::http::experimental::listener::http_listener;
 using web::http::experimental::listener::http_listener_config;
 
-OBECA::RestHandler::RestHandler(const libconfig::Config& cfg, const std::string& url, const unsigned& total_cache_size, const std::map<std::string, std::unique_ptr<OBECA::Service>>& services )
+MBMS_RT::RestHandler::RestHandler(const libconfig::Config& cfg, const std::string& url, const unsigned& total_cache_size, const std::map<std::string, std::unique_ptr<MBMS_RT::Service>>& services )
```

Comment:
The cert values need to be changed to 
````
"/usr/share/5gmag-rt/cert.pem";
 "/usr/share/5gmag-rt/key.pem";
````

Reply:
These are shared between -modem, -gw and -wui.
The API paths need to be adjusted as well.

I'll do the name changes in the other two today too, so we get back to a consistent state.